### PR TITLE
Add Release manager 

### PR DESCRIPTION
**Pull request by AureRita**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | I add the documentation of release manager on quality council  
| Fixed ticket | NA

### CODE REVIEW

File: content/project-organization/quality-council/processes/release-manager.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+title: How to Manage a release
+---
+
+---
```
Comment (nicosomb) [inline content/project-organization/quality-council/processes/release-manager.md:5]: duplicate content?

File: content/project-organization/quality-council/processes/release-manager.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+![Untitled](images/release_manager_img_3.png)
+
+4. Click on Closed (these are the merged PRs for the selected version) 
+
+![Untitled](images/release_manager_img_4.png)
+
+You now know all the PRs in your release. It's highly recommended that you check them all to find out what these PRs might affect. 
+
+
+Translated with www.DeepL.com/Translator (free version)
```
Comment (nicosomb) [inline content/project-organization/quality-council/processes/release-manager.md:41]: you can remove this line

File: content/project-organization/quality-council/processes/release-manager.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+## 1. Know the release
+
+Before even starting to create a ticket in Jira, you need to find out about the release you're going to test. 
+
+To do this, simply : 
+
+1. Go to Prestashop Pull requests: 
+
+![Untitled](images/release_manager_img_1.png)
```
Comment (nicosomb) [inline content/project-organization/quality-council/processes/release-manager.md:24]: please add a title for each picture, instead of "Untitled".

**Review by nicosomb — CHANGES_REQUESTED** [review]

**Comment by AureRita** [comment]
I apply the different changes you asked for

**Review by nicosomb — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by boherm — APPROVED** [review]

File: content/project-organization/quality-council/processes/release-manager.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+title: How to Manage a release
+---
+
+
+# How to create a Test Plan / Release Plan
+
+
+When a new release is made by the developers, a Test Plan and a Release Plan must be created on the QA side. 
+
+The purpose of this page is to explain how to create these different tickets before carrying out a release.
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/release-manager.md:11]:
```suggestion
Although these tickets are currently created on an internal company Jira, the purpose of this documentation is to explain what information is needed to prepare the release, and how to obtain it, before carrying out a release.
```

File: content/project-organization/quality-council/processes/release-manager.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+# How to create a Test Plan / Release Plan
+
+
+When a new release is made by the developers, a Test Plan and a Release Plan must be created on the QA side. 
+
+The purpose of this page is to explain how to create these different tickets before carrying out a release. 
+
+## 1. Know the release
+
+Before even starting to create a ticket in Jira, you need to find out about the release you're going to test.
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/release-manager.md:15]:
```suggestion
The first thing to do is to find out about the release you're going to test. 
```

File: content/project-organization/quality-council/processes/release-manager.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+1. Click on the version you require (for this example, it will be 1.7.8.9) 
+
+![Version](images/release_manager_img_3.png)
+
+4. Click on Closed (these are the merged PRs for the selected version) 
+
+![Closed](images/release_manager_img_4.png)
+
+You now know all the PRs in your release. It's highly recommended that you check them all to find out what these PRs might affect.
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/release-manager.md:35]:
```suggestion
You now have the list of all the PRs merged in your release. It's highly recommended that you check them all to find out what these PRs might affect. 
```

File: content/project-organization/quality-council/processes/release-manager.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+1. Select the various repository affected by the PRs (e.g. a change in product creation) 
+
+![test repos](images/release_manager_img_20.png)
+
+3. Once you've identified the different scenarios that interest you, write them down (or mark them in your notepad). 
+
+![Tests2](images/release_manager_img_21.png)
+
+
+If you've followed all these steps, you should now have your list for Test Plan and Test Execution, which you can share with your team members and developers.
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/release-manager.md:59]:
```suggestion
If you've followed all these steps, you should now have your list of test scenarios which should be tested for this release, which you can share with your team members and developers. 
```

**Review by Robin-Fischer-PS — CHANGES_REQUESTED** [review]
You should delete all the images not used in the documentation :)

**Review by Robin-Fischer-PS — APPROVED** [review]
LGTM :)

**Pull request merged**